doc/nrf/security/ap_protect.rst

```diff
@@ -173,8 +173,20 @@ Based on the available implementation types, you can configure the access port p
 Enabling AP-Protect with :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK`
 ====================================================================
 
-Setting the :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` Kconfig option to ``y`` and compiling the firmware is enough to enable the access port protection mechanism for SoCs of the nRF53 Series and those SoCs of the nRF52 Series that feature the hardware and software type of AP-Protect.
-The access port protection configured in this way cannot be disabled without erasing the flash.
+Setting the :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` Kconfig option to ``y`` and compiling the firmware enables the access protection mechanism for SoCs of the nRF53 Series and those SoCs of the nRF52 Series that feature the hardware and software type of AP-Protect.
+
+Enabling the Kconfig option writes the debugger register in the ``SystemInit()`` function to lock the access port protection in every boot.
+In addition to this, the UICR.APPROTECT register should be written as instructed in :ref:`app_approtect_uicr_approtect`.
+
+.. note::
+    For multi-image builds, this Kconfig option needs to be set for the first image (usually a bootloader).
+    Otherwise, the AP-Protect will not be sufficient and the debugger can be attached to the device.
+
+    When using sysbuild, set the ``SB_CONFIG_APPROTECT_LOCK`` sysbuild Kconfig option, which enables the :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` Kconfig option for all images.
```

Comment:
So if it only needs to be set on the first image with child/parent, why does it need setting on **every** image with sysbuild?

Reply:
This was in line what was done with SB_CONFIG_APPROTECT_USER_HANDLING, which requires the Kconfig values to be set for all the images. Having this work similarly, is easier to understand and it is not harmful.

That said, if you want the sysbuild to only apply the Kconfig for the first image, I can change that.

Please comment if that is what you were after.

Reply:
If it's not an issue then fine to leave it as-is